.github/workflows/japi-compliance-checker.yml

```
@@ -75,6 +84,12 @@ jobs:
           fi
 
       - name: Run japi-compliance-checker
+        if: ${{ inputs.continueOnError }}
+        continue-on-error: true
```

I think we should be able to use the parameter directly
```suggestion
        continue-on-error: ${{ inputs.continueOnError }}
```

Could you please try this?

---

worked

---

Cool :+1: